src/ess/sans/normalization.py

```diff
@@ -413,14 +414,55 @@ def _reduce(da: sc.DataArray) -> sc.DataArray:
     numerator /= denominator.drop_coords(
         [name for name in denominator.coords if name not in denominator.dims]
     )
-    return IofQ[ScatteringRunType](numerator)
+    return numerator
+
+
+def normalize_q(
```

Comment:
I was wondering for all of these if it would be useful to use `functools.wraps` to copy over the docstring of the internal `_normalize` implementation?
But then it would also update the signature, which would replace the input types `FinalSummedQ[ScatteringRunType, Numerator]` with `sc.DataArray`, which is maybe not something we want? (unless there is a way to copy only the docstring over?)

Reply:
Yes, I had the same thought and decided against it.